src/traces/scattergl/convert.js

```diff
@@ -225,6 +225,12 @@ function convertTextStyle(gd, trace) {
     return optsOut;
 }
 
+function weightFallBack(w) {
+    if(w <= 1000) {
+        return w > 500 ? 'bold' : 'normal';
```

Comment:
Shouldn't the threshold be 400?

Reply:
500 is `Medium`  looks very similar to `Regular`.
IMHO it's debatable between 450/500 to 550.

Reply:
Right, ok, I guess if the only options are `normal` or `bold` we have to draw the line somewhere. Maybe just add a comment explaining why 500 is chosen as the threshold.